assets/log-server.ts

```diff
@@ -0,0 +1,80 @@
+import chalk, { Chalk } from 'chalk';
```

Comment:
I'm not sure I think of this file as an "asset"... maybe move back into `cordova-serve`? Then your `.gitignore` cleans up nicely too